.github/workflows/publish.yml

```diff
@@ -0,0 +1,73 @@
+name: Publish
+
+on:
+  push:
+  pull_request:
+  release:
+    types:
+      - published
+
+env:
+  # Stop wasting time caching packages
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+  # Disable sending usage data to Microsoft
+  DOTNET_CLI_TELEMETRY_OPTOUT: true
+  # Project name to pack and publish
+  PROJECT_NAME: Giraffe.OpenApi
+  DOTNET_VERSION: 8.0.301
+  # GitHub Packages Feed settings
+  GITHUB_FEED: https://nuget.pkg.github.com/giraffe-fsharp/
+  GITHUB_USER: dustinmoris
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
👀

Reply:
I believe it would be beneficial to publish it in the same feed as the other Giraffe packages, but I am open to other suggestions as well.